CHANGES.md

```diff
@@ -9,6 +9,7 @@ Change Log
 
 ##### Fixes :wrench:
 * Fixed a bug causing crashes with custom vertex attributes on `Geometry` crossing the IDL. Attributes will be barycentrically interpolated. [#6644](https://github.com/AnalyticalGraphicsInc/cesium/pull/6644)
+* Fixed a bug with loading two Draco models with the same url. [#6668](https://github.com/AnalyticalGraphicsInc/cesium/issues/6668)
```

Comment:
Maybe mention that this also applies to draco encoded i3dm tiles, since that's where most users have run into it.